library/base/src/main/res/values-in/values.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="normalized_app_language">id</string>
```

Comment:
Shouldn't this be "in"

Reply:
It actually doesn't matter because of how the locale object handles the Indonesian language. But `id` is the correct language tag for Indonesian.

Internally the locale object converts Indonesian to `in` due to old behavior and backwards compatibility. This conversion means that older versions of Android need a `values-in` directory, and newer versions prefer `values-id`. To ensure compatibility we just duplicate the correct `values-id` to the incorrect `values-in`